backend/app/views/spree/admin/stock_items/_stock_management.html.erb

```diff
@@ -28,7 +28,9 @@
         <td class="no-padding" rowspan="<%= row_count %>">
           <div class='variant-container'>
             <div class='variant-image'>
-              <%= render 'spree/admin/shared/image', image: variant.display_image(fallback: false), size: :small %>
+              <%= render 'spree/admin/shared/image',
+                image: variant.gallery.images.first.try(:url, :small),
```

Comment:
The thing you're passing into the partial as `image` is actually an `url`. Shouldn't we rename the variable as well?